stubs/extensions/pdo.phpstub

```diff
@@ -111,15 +111,11 @@ class PDO
     public function lastInsertId(?string $name = null) {}
 
     /**
-     * @psalm-taint-sink sql $query
```

Comment:
This is wrong, the query passed to prepare is still a taint sink.

Reply:
Done !